src/Elliptic/Systems/Xcts/Events/ObserveAdmIntegrals.cpp

```diff
@@ -77,6 +81,8 @@ void local_adm_integrals(
         extrinsic_curvature, mesh, boundary_direction);
     const auto face_trace_extrinsic_curvature = dg::project_tensor_to_boundary(
         trace_extrinsic_curvature, mesh, boundary_direction);
+    const auto face_inertial_coords = dg::project_tensor_to_boundary(
+        inertial_coords, mesh, boundary_direction);
```

Comment:
The face coords are available from the DataBox. Use `domain::Tags::Faces<3, domain::Tags::Coordinates<3, Frame::Inertial>>` instead of projecting to the boundary.

Reply:
We don't need the coordinates anymore because we're now using `conformal_face_normal` to get the unit normal vector.